utils/files/get.go

```diff
@@ -69,5 +70,39 @@ func Download(url string, path string) error {
 		return err
 	}
 
+	return nil
+}
+
+// Put puts content on the place of the path
```

Comment:
Is there some check on whether these files are allowed to be deleted? What if a call comes in that says to delete honeytrap.go?

Reply:
Done!

Reply:
Done!